presto-orc/src/test/java/com/facebook/presto/orc/OrcTester.java

```diff
@@ -727,6 +727,12 @@ else if (type == DATE) {
                         return false;
                     }
                 }
+                else if (type == TIMESTAMP) {
+                    if (!filter.testLong(((SqlTimestamp) value).getMillisUtc())) {
```

Comment:
getMillisUtc()  -> getMillis()

Reply:
looks like this is how its being written, https://github.com/prestodb/presto/blob/6b05323bff91fb089b215737676819817da2befb/presto-orc/src/test/java/com/facebook/presto/orc/OrcTester.java#L947
I can make a commit later to remove the usage of the deprecated method. As of now its being used in lot of places which needed to be refactored in a new commit.

Reply:
@bhhari Sounds good. Thanks for explaining.